packages/react-tc-ui-kit/src/components/Button/Button.css

```diff
@@ -1,60 +1,56 @@
 .tc-button {
   position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  border: 1px solid #344675;
-  border-radius: 4px;
-  padding: 10px 32px;
-  min-height: 40px;
-  box-sizing: border-box;
+  margin: 0 auto;
+  outline: none;
+  border: none;
+  border-radius: 15px;
+  padding: 11px 42px;
+  min-width: 137px;
+  min-height: 42px;
   box-shadow: none;
-  font-weight: 600;
-  font-size: 14px;
-  line-height: 1.35;
+  font-size: 18px;
+  line-height: 1.15;
   text-align: center;
-  color: #fff;
-  background-image: linear-gradient(to bottom left, #344675, #263148, #344675);
-  background-position: 0 0;
-  background-size: 100%;
-  background-color: #344675;
+  color: #172f3d;
```

Comment:
i suggest to set default `color: inherit`
also we shoud inherit `font-family, font-weight`